packages/shopify-app-session-storage-sqlite/src/migrations.ts

```diff
@@ -0,0 +1,41 @@
+import {SqliteConnection} from './sqlite-connection';
+
+export const migrationMap = new Map([['internal_v101', migrateToV1_0_1]]);
+
+// need to migrate exisiting scope from varchar 255 to varchar 1024
+async function migrateToV1_0_1(connection: SqliteConnection): Promise<void> {
+  const tempTableName = `${connection.sessionDBIdentifier}_for_migration_toV101`;
+
+  //  1. rename exisiting table
+  const rename = `
```

Comment:
Nit: would it be possible to run this migration in a transaction to prevent data loss?

Reply:
Good point !!!